labelbox/schema/organization.py

```diff
@@ -34,3 +44,109 @@ def __init__(self, *args, **kwargs):
     users = Relationship.ToMany("User", False)
     projects = Relationship.ToMany("Project", True)
     webhooks = Relationship.ToMany("Webhook", False)
+
+    @beta
+    def invite_user(
+            self,
+            email: str,
+            role: Role,
+            project_roles: Optional[List[ProjectRole]] = None) -> Invite:
+        """
+        Invite a new member to the org. This will send the user an email invite
+
+        Args:
+            email (str): email address of the user to invite
+            role (Role): Role to assign to the user
+            project_roles (Optional[List[ProjectRoles]]): List of project roles to assign to the User (if they have a project based org role).
+
+        Returns:
+            Invite for the user
+
+        Creates or updates users. This function shouldn't directly be called. 
+        Use `Organization.invite_user`
+
+        Note: that there is a really unclear foreign key error if you use an unsupported role.
+        - This means that the `Roles` object is not getting the right ids
+        """
+        remaining_invites = self.invite_limit().remaining
+        if remaining_invites == 0:
+            raise LabelboxError(
+                "Invite(s) cannot be sent because you do not have enough available seats in your organization. "
+                "Please upgrade your account, revoke pending invitations or remove other users."
+            )
+
+        if not isinstance(role, Role):
+            raise TypeError(f"role must be Role type. Found {role}")
```

Comment:
This often helps me. I find myself passing ids instead of objects. It might not be pythonic but I think that it adds value. Especially when users feel overwhelmed by the SDK, it is nice to make the errors super clear.

Reply:
Fixed.